### PR TITLE
Fixes issue #1

### PR DESCRIPTION
**Pull request by jancinert**

### CODE REVIEW

File: src/Universal/Query/CreateTableQuery.php [hunk 1]
```diff
@@ -45,9 +45,11 @@ public function engine($engine)
         return $this;
     }
 
-    public function column($name)
+    public function column($col)
     {
-        $col = new Column($name);
+        if (gettype($col) == 'string') {
```
Comment (c9s) [inline src/Universal/Query/CreateTableQuery.php]: is_string is faster

**Review by c9s — CHANGES_REQUESTED** [review]

**Review by jancinert — COMMENTED** [review]
You're right.

File: src/Universal/Query/CreateTableQuery.php (hunk 1 above)
Comment (jancinert) [reply src/Universal/Query/CreateTableQuery.php]: Done

**Comment by jancinert** [comment]
Is anything to do to resolve the PR?

**Comment by c9s** [comment]
Nope thanks!

**Pull request merged**